libraries/Microsoft.Bot.Builder.Azure.Blobs/BlobsTranscriptStore.cs

```diff
@@ -98,7 +119,7 @@ public async Task LogActivityAsync(IActivity activity)
                             updatedActivity.Type = ActivityTypes.Message; // fixup original type (should be Message)
                             updatedActivity.LocalTimestamp = activityAndBlob.Item1.LocalTimestamp;
                             updatedActivity.Timestamp = activityAndBlob.Item1.Timestamp;
-                            await LogActivityToBlobClientAsync(updatedActivity, activityAndBlob.Item2, true).ConfigureAwait(false);
```

Comment:
Ah, missed that one. I guess it depends. It looks like our previous flow would only update on `ActivityTypes.MessageUpdate` and would otherwise write a new one. Given that the `blobName` includes both the `activity.Id` along with a timestamp, I _think_ this is a non-issue??

Unfortunately, there's no way to specify both `overwrite` and `BlobUploadOptions`, however [`Conditions`](https://docs.microsoft.com/en-us/dotnet/api/azure.storage.blobs.models.blobuploadoptions.conditions?view=azure-dotnet#Azure_Storage_Blobs_Models_BlobUploadOptions_Conditions) exists on  `BlobUploadOptions`, but I'm not sure how best to implement them.

Thoughts?

Reply:
@EricDahlvang RE: this comment thread, how's [this change](https://github.com/microsoft/botbuilder-dotnet/pull/5303/commits/a3a5ed240ad115bf19c2329a664f11a400ca910b)?

Reply:
Can we add a test, to ensure behavior has not changed related to overwrite?

Reply:
Will do. Will revisit this PR soon.